multisite/models.py

```diff
@@ -74,7 +78,7 @@ def _expand_netloc(cls, host, port=None):
             bits = host.split('.')
 
         result = []
-        for i in xrange(0, (len(bits) + 1)):
+        for i in range(0, (len(bits) + 1)):
```

Comment:
Doesn't this change behaviour in Python 2.7? Is there a way to fix this without changing behaviour?